rdrf/rdrf/services/io/defs/migrations_list.py

```diff
@@ -0,0 +1,285 @@
+MIGRATIONS_LIST = {
```

Comment:
I was wondering whether we should name this something a bit more descriptive of its purpose e.g. `MIGRATIONS_CONSIDERED_FOR_IMPORT_EXPORT`, and file name updated accordingly. Just thinking if someone new stumbles across this data structure, it would help  in understanding the purpose of the code.
And also maybe a brief comment that refers to your `docs/import-export-logic-updates.md`.

Reply:
+1 And also change the name in the django check's error